content/docs/forms.md

```diff
@@ -289,10 +306,13 @@ setTimeout(function() {
 
 ```
 
-## Alternatives to Controlled Components {#alternatives-to-controlled-components}
+## Alternatieven Voor Gecontroleerde Componenten {#alternatives-to-controlled-components}
 
-It can sometimes be tedious to use controlled components, because you need to write an event handler for every way your data can change and pipe all of the input state through a React component. This can become particularly annoying when you are converting a preexisting codebase to React, or integrating a React application with a non-React library. In these situations, you might want to check out [uncontrolled components](/docs/uncontrolled-components.html), an alternative technique for implementing input forms.
+Het kan soms vervelend zijn om gecontroleerde componenten te gebruiken, omdat je een event handler moet schrijven voor elke manier waarop je data kan veranderen en alle input state doorheen het React component moet sturen.
```

Comment:
```suggestion
Het kan soms vermoeiend zijn om aangestuurde componenten te gebruiken, omdat je een event handler moet schrijven voor elke manier waarop jouw data kan veranderen en alle input staat door het React component heen moet sturen.
```

Tedious was probably meant as monotone/fatiguing/dull. In this case `vermoeiend` is more appropriate.